src/lamp_py/performance_manager/l0_rt_trip_updates.py

```diff
@@ -35,19 +37,19 @@ def get_tu_dataframe_chunks(
         "trip_update.vehicle.id",
         "trip_update.trip.trip_id",
     ]
-    trip_update_filters = [
-        ("trip_update.trip.direction_id", "in", (0, 1)),
-        ("trip_update.trip.trip_id", "!=", "None"),
-        ("trip_update.vehicle.id", "!=", "None"),
-        ("trip_update.trip.route_id", "in", route_ids),
-        ("trip_update.stop_time_update.arrival.time", ">", 0),
-    ]
+    trip_update_filters = (
+        (pc.field("trip_update.trip.direction_id").isin((0, 1)))
+        & (pc.field("trip_update.trip.trip_id").is_valid())
+        & (pc.field("trip_update.vehicle.id").is_valid())
+        & (pc.field("trip_update.trip.route_id").isin(route_ids))
+        & (pc.field("trip_update.stop_time_update.arrival.time") > 0)
+    )
 
     # 100_000 batch size should result in ~5-6 GB of memory use per batch
     # of trip update records
     return read_parquet_chunks(
         to_load,
-        max_rows=250_000,
+        max_rows=1_000_000,
```

Comment:
Testing this locally, memory usage never got above ~800MB for loading a trip_updates file. Now that we are not exploding these in pandas resource usage is significantly smaller.